kernel/univ.ml

```diff
@@ -963,6 +963,15 @@ struct
 
 end
 
+type 'a univ_abstracted = {
+  univ_abstracted : 'a;
+  bound_univs : Names.Name.t array
+}
```

Comment:
You could also use the conventional name of "closure". 

Also, at first, I did not get that the projection `univ_abstracted` meant the body. So, maybe `univ_closure_body` and `univ_closure_binders`?

Reply:
Wouldn't a closure contain values for the universes rather than binder names?

Reply:
> Wouldn't a closure contain values for the universes rather than binder names?

You're right. My confusion.

I still feel that the current terminology can be made more appealing though. So, maybe `univ_abstracted_body` and `univ_abstracted_binders`. Or `abstraction` in place of `abstracted`. Of course, that's pretty subjective, so maybe other opinions would help to move from subjective to more objective.